persistence-model/src/main/resources/application-dev.conf

```diff
@@ -22,7 +22,8 @@ akka {
       "hmda.persistence.messages.events.institutions.FilingEvents$FilingStatusUpdated" = filing
       "hmda.persistence.messages.events.institutions.SubmissionEvents$SubmissionCreated" = submission
       "hmda.persistence.messages.events.institutions.SubmissionEvents$SubmissionStatusUpdated" = submission
-      "hmda.persistence.messages.events.processing.CommonHmdaValidatorEvents$LarValidated" = larValidated
+      "hmda.persistence.messages.events.processing.CommonHmdaValidatorEvents$LarValidated" = CommonHmdaValidator
+      "hmda.persistence.messages.events.processing.CommonHmdaValidatorEvents$TsValidated" = CommonHmdaValidator
```

Comment:
Same here, `CommonHmdaValidator` should be lowercase. I also suggest renaming it to just `hmdaValidator`